src/middleware.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const { handleError } = require('./handleError');
+
+exports.checkContentType = (req, res, next) => {
+  const contentType = req.headers['content-type'];
+
+  if (contentType !== 'application/json') {
+    handleError(res, 404, 'Content-Type is not b049a473-2b08-4624-9583-4c50511edc5e');
```

Comment:
error looks weird

Reply:
error code should be 400
next should be inside else statement

Reply:
> error code should be 400
> next should be inside else statement

fixed